rasa/core/policies/embedding_policy.py

```diff
@@ -556,8 +560,17 @@ def predict_action_probabilities(
         tf_feed_dict = self.tf_feed_dict_for_prediction(tracker, domain)
 
         confidence = self.session.run(self.pred_confidence, feed_dict=tf_feed_dict)
+        confidence = confidence[0, -1, :].tolist()
 
-        return confidence[0, -1, :].tolist()
+        # normalise scores if turned on
+        if self.loss_type == "softmax" and self.normalize_top_num_actions > 0:
```

Comment:
I don't think we persist config in core

Reply:
You're right, we didn't -- took me a little while to realize that 😃hence the edits to persistence + loading on L588-591 and 673, 685

Reply:
but `loss_type` also is not persisted

Reply:
oh I see, good point. I guess I didn't catch that because it is the default. Makes sense then to persist it bc it's used at runtime right? can add it